### ISSUE TITLE
Model Analyzer fails for Torchscript Model

### ISSUE THREAD

**Issue by eycheung** [triton-inference-server/model_analyzer#622]
**Description**
When calling the model analyzer, I get the following issue.
```
Request concurrency: 1
Failed to maintain requested inference load. Worker thread(s) failed to generate concurrent requests.
Thread [0] had error: PyTorch execute failure: The following operation failed in the TorchScript interpreter.
Traceback of TorchScript, serialized code (most recent call last):
  File "code/__torch__/___torch_mangle_6127.py", line 15, in forward
    attention_mask0 = torch.to(attention_mask, dtype=4, layout=0, device=torch.device("cuda"))
    input_ids1 = torch.to(decoder_input_ids, dtype=4, layout=0, device=torch.device("cuda"))
    _0 = (model).forward(input_ids0, attention_mask0, input_ids1, )
          ~~~~~~~~~~~~~~ <--- HERE
    return _0
  File "code/__torch__/transformers/models/t5/modeling_t5/___torch_mangle_6126.py", l
```

**Triton Information**
Version: **23.01**

**To Reproduce**
1. Take a public T5 Model from HuggingFace (https://huggingface.co/docs/transformers/model_doc/t5)
2. Create a wrapper for the T5 model to only return the logits.
```python
from transformers import AutoModelForSeq2SeqLM

class PyTorchToTorchScript(torch.nn.Module):
    def __init__(self):
        super(PyTorchToTorchScript, self).__init__()
        self.model = AutoModelForSeq2SeqLM.from_pretrained(PATH_TO_MODEL_FILES).eval().cuda().half()
    def forward(self, input_ids, attention_mask, decoder_input_ids):
        return self.model(input_ids=input_ids.cuda(), attention_mask=attention_mask.cuda(), decoder_input_ids=decoder_input_ids.cuda())[0]
```
3. Create a torchscript model by tracing through the wrapped model code.
```python
traced_script_module = torch.jit.trace(
    pt_model, 
    (input_ids.cuda(), attention_mask.cuda(), decoder_input_ids.cuda())
)
traced_script_module.save(os.path.join(PATH_TO_MODEL_FILES, "model.pt"))

# NOTE: the loaded model file works
pt_model_loaded = torch.jit.load(os.path.join(PATH_TO_MODEL_FILES, "model.pt"))
pt_model_loaded.forward(input_ids.cuda(), attention_mask.cuda(), decoder_input_ids.cuda())
```
4. Move `model.pt` to model repository.
5. Create `config.pbtxt`

```
name: "torchscript"
platform: "pytorch_libtorch"
max_batch_size: 128
input [
        {
                        name: "input_ids__0"
                        data_type: TYPE_INT64
                        dims: [ 60 ]
        },
        {
                        name: "attention_mask__1"
                        data_type:  TYPE_INT64
                        dims: [ 60 ]
        },
        {
                        name: "decoder_input_ids__2"
                        data_type:  TYPE_INT64
                        dims: [ 60 ]
        }
]
output [
        {
                        name: "logits__0"
                        data_type:  TYPE_FP16
                        dims: [ 60, 32100]
        }
]
instance_group [
    {
      count: 1
      kind: KIND_GPU
    }
]
```
6. Start docker image for model analyzer
```bash
docker run -it --gpus all \
      -v /var/run/docker.sock:/var/run/docker.sock \
      -v <PATH_TO_MODEL_REPO>:<PATH_TO_MODEL_REPO> \
      -v <PATH_TO_MODEL_REPO>/outputs:<PATH_TO_MODEL_REPO>/outputs \
      --net=host nvcr.io/nvidia/tritonserver:23.01-py3-sdk
```
7. Run model analyzer in container. (fails with both local triton server and docker image triton server)
```bash
model-analyzer profile \
    --model-repository <PATH_TO_MODEL_REPO> \
    --profile-models torchscript --triton-launch-mode=docker \
    --output-model-repository-path $<PATH_TO_MODEL_REPO>/outputs \
    --export-path profile_results
```

Any help would be appreciated in debugging this!

**Comment by eycheung** [comment]
I'll also add that running the server itself seems to work:
```python
import numpy as np
import sys

import tritonclient.grpc as grpcclient


try:
    keepalive_options = grpcclient.KeepAliveOptions(
        keepalive_time_ms=2**31 - 1,
        keepalive_timeout_ms=20000,
        keepalive_permit_without_calls=False,
        http2_max_pings_without_data=2
    )
    triton_client = grpcclient.InferenceServerClient(
        url='localhost:8001',
        verbose=False,
        keepalive_options=keepalive_options)
except Exception as e:
    print("channel creation failed: " + str(e))
    sys.exit()

model_name = "torchscript"
# Infer
inputs = []
outputs = []
inputs.append(grpcclient.InferInput('input_ids__0', [1, 60], "INT64"))
inputs.append(grpcclient.InferInput('attention_mask__1', [1, 60], "INT64"))
inputs.append(grpcclient.InferInput('decoder_input_ids__2', [1, 60], "INT64"))

input0_data = np.random.randint(9999, size=(1, 60)).astype(np.int64)
input1_data = np.random.randint(1, size=(1,60)).astype(np.int64)
input2_data = np.random.randint(9999, size=(1,60)).astype(np.int64)

# Initialize the data
inputs[0].set_data_from_numpy(input0_data)
inputs[1].set_data_from_numpy(input1_data)
inputs[2].set_data_from_numpy(input2_data)

outputs.append(grpcclient.InferRequestedOutput('logits__0'))

# Test with outputs
results = triton_client.infer(model_name=model_name,
                                inputs=inputs,
                                outputs=outputs,
                                headers={'test': '1'})

# Get the output arrays from the results
output0_data = results.as_numpy('logits__0')
print(output0_data)
```

**Comment by eycheung** [comment]
Ok I have found the issue is with the inputs since the default `input-data` is random and not suited for representing the `attention_mask`. The fix is to just initialize the data as zeros.
